introduction.md

```diff
@@ -1,3 +1,7 @@
 hi i just want to gegt past this lesson so jeres a lot ofngjikgfmhnrjn
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?